index.js

```diff
@@ -4,15 +4,52 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "green"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+let headerTwo = document.querySelector("h2")
+headerTwo.innertext = traveler.name  
+
+let em = document.querySelector("em") 
+em.innerText = traveler.nickname  
+
+image = document.querySelector("img")
+image.src = "pics/raffy.jpg" 
+image.alt = "raffy"  
```

Comment:
You should be using the data from 'traveler', not hard coding the image src and alt